stories/modal.stories.js

```diff
@@ -86,6 +90,21 @@ class ModalStory extends React.Component {
   }
 }
 
+ModalStory.propTypes = {
+  maxHeight: PropTypes.string.isRequired,
+  numberOfLines: PropTypes.number.isRequired,
+  width: PropTypes.string.isRequired
+};
+
 const stories = storiesOf('Modal', module);
 stories.addDecorator(withKnobs);
-stories.add('default', () => <ModalStory p="large">Foo</ModalStory>);
+stories.add('default', () => (
+  <ModalStory
+    p="large"
+    numberOfLines={number('Lines of Content', 100)}
+    maxHeight={text('Max Height', '100%')}
+    width={text('Modal Width', '500px')}
```

Comment:
AHA!

Reply:
I think having the knob inside of the component itself was preventing it from re-rendering. We need it outside of the component to pass it in as a prop to trigger a re-render.